node/src/components/consensus/protocols/highway/tests.rs

```diff
@@ -66,7 +73,7 @@ pub(crate) fn new_test_highway_protocol() -> Box<dyn ConsensusProtocol<NodeId, C
 fn test_highway_protocol_handle_message_parse_error() {
     // Build a highway_protocol for instrumentation
     let mut highway_protocol: Box<dyn ConsensusProtocol<NodeId, ClContext>> =
-        new_test_highway_protocol();
+        new_test_highway_protocol(vec![]);
```

Comment:
Shouldn't we leave Bob in this list, to leave the existing tests unchanged?

Reply:
Maybe. I didn't see anything in the existing tests that would require `BOB` to be banned thoguh.